scss/components/header/_navigation.scss

```diff
@@ -1,53 +1,222 @@
-                background: $bg-line-navigation-item;
-                transition: .2s width;
+        &:after {
+            content: '';
+            position: absolute;
+            left: 0;
+            bottom: 0;
+            right: 0;
+            width: 0;
+            height: 1px;
+            background: $bg-line-navigation-item;
+            transition: .2s width;
 
+        }
+
+        &.logo {
+            margin: 20px -13px 0 14px;
+        }
+    }
+
+    &.active,
+    &:hover {
+
+        a {
+            &:not(.logo) {
+                &:after {
+                    width: $width-line-navigation-item-active;
+                }
             }
 
             &.logo {
-                margin: 20px -13px 0 14px;
+                opacity: .8;
             }
+        }
+    }
+}
 
+@include screen-x-large {
+    .navigation {
+        .navigation-item{
+            margin: 0 1vw 10px;
+        }
 
+        .logo {
+            margin: 20px -1vw 0 -1vw;
         }
+    }
+}
 
-        &.active,
-        &:hover {
+@include screen-large {
+    .navigation {
+        flex-flow: row wrap;
+        justify-content: center;
+        height: inherit;
+        padding: 30px 0;
+    }
 
-            a {
-                &:not(.logo) {
-                    &:after {
-                        width: $width-line-navigation-item-active;
-                    }
+    .navigation-item-logo {
+        order: -1;
+
+        a.logo {
+            margin: 0 10px 0 0;
+
+            img {
+                margin-bottom: 0;
+            }
+        }
+    }
+}
+
+.burger-menu {
+    display: none;
+
+    .burger-shower {
+        display: none;
+    }
+}
+
+.icon-burger {
+    position: fixed;
+    z-index: 2;
+    right: 0;
+    top: 0;
+    cursor: pointer;
+    -webkit-tap-highlight-color: transparent;
+    transition: transform 400ms;
+    -moz-user-select: none;
+    -webkit-user-select: none;
+    -ms-user-select: none;
+    user-select: none;
+    
+    .line-burger {
+      fill:none;
+      transition: stroke-dasharray 400ms, stroke-dashoffset 400ms;
+      stroke:#fff;
+      stroke-width:5.5;
+      stroke-linecap:round;
+      
+      &.line-top {
+        stroke-dasharray: 40 160;
+      }
+      
+      &.line-middle {
+        stroke-dasharray: 40 142;
+        transform-origin: 50%;
+        transition: transform 400ms;
+      }
+      
+      &.line-bottom {
+        stroke-dasharray: 40 85;
+        transform-origin: 50%;
+        transition: transform 400ms, stroke-dashoffset 400ms;
+      }
+    }
+  }
+
+@include screen-medium {
+    .navigation {
+        padding: 10px 0;
+
+        &:not(.navigation-burger) {
+            padding-top: 18px;
+
+            .navigation-item {
+                margin: 0;
+
+                a {
+                    padding: 0;
                 }
+            }
+        }
+    }
 
-                &.logo {
-                    opacity: .8;
+    .burger-menu {
+        float: right;
```

Comment:
It seems like you forgot clearfix for parent

Reply:
`float: right` is a dead code. delete